Software 0.32.5 release notes + versioning note (#3145)
astronomer/docs: software/release-notes.md

0.33 is the latest stable version of Astronomer Software, while 0.32 remains the latest long-term support (LTS) release. To upgrade to 0.33, see [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, see [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](https://docs.astronomer.io/astro/cli/release-notes).

> :::info 

Author: Now that we include all versions on the same page, I thought I needed to clarify why there's multiple release notes for the same change across versions. If there's an industry-specific term for explaining this release flow, please lmk

    - [CVE-2023-37920](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-37920)
    - [CVE-2023-35945](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-35945)

## 0.32.5

Release date: December 8, 2023

### Additional improvements

- You can now configure a global index name prefix to use for both Fluentd and sidecar-based logging, which allows Astronomer Software to retain logs when you migrate between logging solutions. To enable this feature, add the following lines to your `config.yaml` file:

    ```yaml
    global:
      logging:
        indexNamePrefix: <your-index-prefix>
    ```
- Deprecated Kubernetes 1.23.
- Astronomer Software now throws an error if you attempt to install it with an unsupported version of Kubernetes.

> - You can now set the following configuration to bypass proxy support when Astronomer Software redirects to Azure login. This might be required in some circumstances when importing identity provider groups as Teams: <!-- https://github.com/astronomer/issues/issues/5844 -->

Author: Also drafted a Zendesk topic for this, please review: https://astronomer.zendesk.com/knowledge/editor/01HH232T9717CRGS52PTN1T1M2/en-us?brand_id=360001488713

### Additional improvements

- You can now configure a global index name prefix to use for both [Fluentd and sidecar-based logging](export-task-logs.md), which allows Astronomer Software to retain logs when you migrate between logging solutions. To enable this feature, add the following lines to your `config.yaml` file:

    ```yaml
    global:
      logging:
        indexNamePrefix: <your-index-prefix>
    ```
    
- Deprecated Kubernetes 1.23.
- Astronomer Software now throws an error if you attempt to install it with an unsupported version of Kubernetes.
- You can now set the following configuration to bypass proxy support when Astronomer Software redirects to Azure login. This might be required in some circumstances when importing identity provider groups as Teams: <!-- https://github.com/astronomer/issues/issues/5844 -->

    ```yaml
    astronomer:
      houston:
        config:
          auth:
            microsoft:
              useExternalProxy: false
    ```

### Bug fixes

- Fixed an issue where deleting a user from Astronomer Software without deleting the user from an Idp Team could result in the user not being active if they're re-added.

> - Fixed an issue where you could create users with the Houston API when SCIM was enabled and `userManagement.enable` was `false`.

Reviewer: Fixed an issue that allowed user creation with the Houston API when SCIM was enabled and `userManagement.enable` was `false`

> - Fixed an issue where you could create users with the Houston API when SCIM was enabled and `userManagement.enable` was `false`.

Reviewer 2: ```suggestion
- Fixed an issue that allowed user creation with the Houston API when SCIM was enabled and `userManagement.enable` was `false`.
```
